src/components/playerstats/playerstats.js

```diff
@@ -274,17 +274,55 @@ function getMediaSourceStats(session, player) {
         });
     }
 
+    const frameRate = videoStream.AverageFrameRate || videoStream.RealFrameRate;
+    if (frameRate) {
+        sessionStats.push({
+            label: globalize.translate('Framerate'),
+            value: frameRate
+        });
+    }
+
     if (videoStream.BitRate) {
         sessionStats.push({
             label: globalize.translate('LabelVideoBitrate'),
             value: getDisplayBitrate(videoStream.BitRate)
         });
     }
 
+    if (videoStream.BitDepth) {
```

Comment:
`BitDepth` is useless here because `PixelFormat` and even `Profile` already provide the depth value.

```
yuv4xxp -> [8] <- Main
yuv4xxp10 -> [10] <- Main10
```

Reply:
My plan is to utilize horizontal space, rather than vertically expanding the items.
This can prevent the overlay from extending beyond the screen on mobile devices.

<img width="520" height="177" alt="osd" src="https://github.com/user-attachments/assets/0696ad6d-a86f-4fa2-ba2c-db0c8bf13549" />

Reply:
Looks good to me! :+1: